### PR TITLE
[#147170299] Adjust timeout and grace period to exceed 30 days

### PR DESCRIPTION
**Pull request by charlieoduk**
**What does the PR do**
- Updated check.js to allow for timeout and grace period exceeding 30 days
- Removed CSS formatting from my_checks_desktop.html to display the modal headings on the left

**Related Pivotal Tracker story**
#147170299

### CODE REVIEW

File: static/js/checks.js [hunk 1]
```diff
@@ -35,16 +35,17 @@ $(function () {
...
+            '40%': 43200,
+            '60%': 604800,
+            '80%': (2592000 * 6),
+            'max': (2592000 * 12),
         },
         pips: {
             mode: 'values',
-            values: [60, 1800, 3600, 43200, 86400, 604800, 2592000],
-            density: 4,
+            values: [60, 3600, 86400, 2592000, (2592000 * 6), (2592000 * 12)],
```
Comment (amwaleh) [inline static/js/checks.js]: please convert `2592000` into a constant to adhere to **DRY** principal

File: static/js/checks.js [hunk 2]
```diff
@@ -64,16 +65,17 @@ $(function () {
...
```
Comment (amwaleh) [inline static/js/checks.js]: convert (2592000 * 6) and (2592000 * 12) into constants, this will enable changing of the value to be done in one place

**Review by amwaleh — CHANGES_REQUESTED** [review]
Niceeee !! work, please attend to the comments

File: static/js/checks.js (hunk 2 above)
Comment (SProjects) [reply static/js/checks.js]: The code has been DRYed.

**Review by amwaleh — DISMISSED** [review]
Rebase the branch

**Review by amwaleh — DISMISSED** [review]

**Comment by SProjects** [comment]
@andela-amwaleh The conflict has been removed. I think the branch is ready to merge.

**Review by amwaleh — APPROVED** [review]

**Pull request merged**